.github/workflows/handle-new-external-pr.yml

```
@@ -54,6 +54,7 @@ jobs:
           LABEL: "Contribution Form Filled"
 
       - name: Send Notification
+        if: startsWith(github.event.pull_request.title, 'Test |') != true
```

```suggestion
        if: startsWith(github.event.pull_request.title, 'Test') != true
```
I think that it will be better if we will check only test.

---

https://github.com/demisto/content/pull/34788/commits/58e748db31f0109d4e48ec9357035d42e31e2746